src/runtime/index.tsx

```diff
@@ -16,9 +16,10 @@ export const Runtime: React.FC = () => {
     }
 
     try {
+        const projectName = 'unknown_project';
```

Comment:
I do not know how to get the project name correctly so that 2 documents on the same domain cannot use tokens.